test/any/parsing/BaseParser-test.js

```diff
@@ -0,0 +1,55 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+describe('BaseParser', function() {
+  beforeEach(function() {
+    foam.CLASS({
+      name: 'TestParser',
+      extends: 'org.chromium.webidl.BaseParser',
+
+      constants: {
+        PARSE_COMPLETE: 'Parse complete',
+        PARSE_INCOMPLETE: 'Parse incomplete'
+      },
+
+      properties: [
+        {
+          name: 'separator',
+          factory: function() {
+            return this.Parsers.create().literal(' ');
+          }
+        }
+      ],
+
+      methods: [
+        function symbolsFactory() {
+          return Object.assign(this.SUPER(), foam.Function.withArgs(
+              function(literal) {
+                return {
+                  START: literal('interface Frobinator;')
+                }
+              },
+              this.Parsers.create({ separator: this.separator }),
+              this));
+        },
+        function actionsFactory() {
```

Comment:
Leave this out. The base class has this implementation. Unless, are you worried that the base class may later choose a different implementation and you want to squash it?